Readability.js

```diff
@@ -552,7 +552,7 @@ Readability.prototype = {
    * whitespace in between. If the given node is an element, the same node is
    * returned.
    */
-  _nextElement: function (node) {
```

Comment:
The comment above the method definition should also reflect the change.

Reply:
I have made the necessary change to the the docs. Do let me know if it looks fine or do I need to make another change.